sdk/nodejs/storage/zMixins.ts

```diff
@@ -321,39 +319,46 @@ export interface QueueContext extends appservice.Context<void> {
  */
 export interface QueueHostExtensions {
     /** The maximum interval between queue polls. Minimum is 00:00:00.100 (100 ms). */
-    maxPollingInterval?: string,
+    maxPollingInterval?: string;
 
     /** The time interval between retries when processing of a message fails. */
-    visibilityTimeout?: string,
+    visibilityTimeout?: string;
 
     /** The number of queue messages that the Functions runtime retrieves simultaneously and processes in parallel. */
-    batchSize?: number,
+    batchSize?: number;
 
     /** The number of times to try processing a message before moving it to the poison queue. */
-    maxDequeueCount?: number,
+    maxDequeueCount?: number;
 
     /** Whenever the number of messages being processed concurrently gets down to this number, the runtime retrieves another batch. */
-    newBatchThreshold?: number,
+    newBatchThreshold?: number;
 }
 export interface QueueHostSettings extends appservice.HostSettings {
     extensions?: {
         queues: QueueHostExtensions,
-    }
+    };
 }
 
 /**
  * Signature of the callback that can receive queue notifications.
  */
-export type QueueCallback = appservice.Callback<QueueContext, Buffer, void>;
+export type QueueCallback = appservice.Callback<QueueContext, any, void>;
 
-export interface QueueFunctionArgs extends appservice.CallbackArgs<QueueContext, Buffer, void> {
+export interface QueueFunctionArgs extends appservice.CallbackArgs<QueueContext, any, void> {

```

Comment:
should this be typed as `Buffer | string | object`?

Reply:
how is this helpful? (honest question)

Reply:
it makes the callback clearer about the tyeps of data you might expect to get.  i.e. `any` literally means: you could get anything back (like a boolean), at least with the union type we're being explicit that you should at least expect to get one of these types back.

Just a thought :)

Reply:
So, I gave it a try: sending a message "true" results in a `boolean` argument (printed `typeof`), "1" becomes a `number`. They are valid JSON...

Reply:
Ahhhh.  Fascinating!  Ok `any` it is.  We might want to mention that JSON behavior more explicitly (i.e. that even simple strings can be converted to JS values, not just objects/arrays).  But i don't care that much :)